package.json

```diff
@@ -23,6 +23,8 @@
     "unit": "nyc --check-coverage mocha --ui tdd --check-leaks"
   },
   "dependencies": {
+    "chalk": "^4.0.0",
+    "colors": "^1.4.0",
```

Comment:
Looks like you accidentally left colors installed here. Other than that, looks good; will take it for a spin when this is fixed.

Reply:
Ok - I've removed the colors module. GOod catch.